ydl_podcast/__init__.py

```diff
@@ -193,25 +200,37 @@ def cleanup(sub):
 
 def write_xml(sub):
     directory = os.path.join(sub['output_dir'], sub['name'])
+
+    image_tag = ""
+
+    if sub['image'] is not None:
+        image_url = "/".join([sub["url_root"], sub["name"], sub["image"]])
+        image_tag = "<image><url>%s</url></image>" % (image_url)
+
     xml = """<?xml version="1.0"?>
-            <rss version="2.0" xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd">
-            <channel>
-            <updated>%s</updated>
-            <title>%s</title>
-            <link href="%s" rel="self" type="application/rss+xml"/>""" \
-                    % (datetime.datetime.now(),
-                       sub['name'],
-                       '/'.join([sub['url_root'], "%s.xml" % sub['name']]))
+             <rss version="2.0" xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd">
+             <channel>
+             <updated>%s</updated>
+             <title><![CDATA[%s]]></title>
+             %s
+             <link href="%s" />
+          """ % (datetime.datetime.now(),
+                 sub['title'] or sub['name'],
```

Comment:
need to use sub.get('title') here too, to avoid `KeyError` if the user has not defined a title.